conf.yaml

```
@@ -1128,8 +1128,8 @@ contents:
                 private:    1
                 single:     1
                 current:    *stackcurrent
-                branches:   [ {main: master}, 8.4, 8.3, 8.2, 8.1 ]
-                live:       [ main, 8.4 ]
+                branches:   [ {main: master}, 8.5, 8.4, 8.3, 8.2, 8.1 ]
+                live:       [ main, 8.5 ]
```

Per earlier suggestions, I think it's too soon to drop 8.4 from the "live" lists:
```suggestion
                live:       [ main, 8.4 ]
```